pkg/grpc_testing/stub_server.go

```diff
@@ -93,20 +94,27 @@ func (ss *StubServer) Addr() string {
 
 type dummyStubServer struct {
 	testpb.UnimplementedTestServiceServer
-	body []byte
+	body    []byte
+	counter int
 }
 
-func (d dummyStubServer) UnaryCall(context.Context, *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+func (d *dummyStubServer) UnaryCall(context.Context, *testpb.SimpleRequest) (*testpb.SimpleResponse, error) {
+	d.counter++
```

Comment:
don't know if we do parallel requests in our testing, but maybe do an atomic increment? that way people don't have to think about this going forward

Reply:
Makes sense, updated :)